src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -109,9 +109,11 @@ public class BlockStoreQueue : IBlockStore, IDisposable
         /// </para>
         /// </summary>
         public async Task InitializeAsync()
-        {
+        {             
```

Comment:
Boo

Reply:
Yeah wtf who did that? 🙄

Reply:
Ah @noescape00 fixed it already thanks.

Reply:
>Yeah wtf who did that?

Lol, you did 😂 

https://github.com/stratisproject/StratisBitcoinFullNode/pull/1726/commits/aa32f674a3860072e769350216901ffd76fd61d8

Reply:
@dangershony there is a tool for VS: Trailing Whitespace Visualizer
It shows trailing whitespaces and removes them when you hit CTRL+S